docs/pages/versions/v34.0.0/sdk/mail-composer.md

```diff
@@ -16,7 +16,7 @@ import * as MailComposer from 'expo-mail-composer';
 
 ### `MailComposer.composeAsync(options)`
 
-Opens a mail modal for iOS and a mail app intent for Android and fills the fields with provided data.
+Opens a mail modal for iOS and a mail app intent for Android and fills the fields with provided data. On iOS you will need to be signed into the Mail app.
```

Comment:
For this change to be persisted in future SDKs, could you also make it in `docs/pages/versions/unversioned/sdk/mail-composer.md`?